tests/unit/SchemaForm.spec.js

```diff
@@ -17,6 +17,40 @@ markRaw(FormSelect)
 markRaw(FormText)
 markRaw(SchemaForm)
 
+describe('schemaRowClasses', () => {
+  it('renders form with additional classes', () => {
+    const schema = {
+      firstName: {
+        component: FormText,
+        label: 'First Name'
+      },
+      lastName: {
+        component: FormText,
+        label: 'Last Name'
+      }
+    }
+    const wrapper = mount(SchemaForm, {
+      props: { schema, modelValue: {}, schemaRowClasses: 'custom-class-a' }
+    })
+    expect(wrapper.find('.schema-row + .custom-class-a').exists()).toBe(true)
+  })
+  it('renders form with default class', () => {
+    const schema = {
+      firstName: {
+        component: FormText,
+        label: 'First Name'
+      },
+      lastName: {
+        component: FormText,
+        label: 'Last Name'
+      }
+    }
+    const wrapper = mount(SchemaForm, {
+      props: { schema, modelValue: {}, schemaRowClasses: '' }
+    })
+    expect(wrapper.find('.schema-row').exists()).toBe(true)
+  })
+})
```

Comment:
Please remove all this and insert the following after the closing `})` for the `it renders a form based on a schema` block

```
describe('schemaRowClasses prop', () => {
    it('renders form rows with user defined classes', () => {
      const schema = {
        firstName: {
          component: FormText,
          label: 'First Name'
        },
        lastName: {
          component: FormText,
          label: 'Last Name'
        }
      }
      
      const wrapper = mount(SchemaForm, {
        props: { schema, modelValue: {}, schemaRowClasses: 'custom-class-a' }
      })
      
      expect(wrapper.findAll('.schema-row + .custom-class-a').length()).toEqual(2)
    })
  })
```

Reply:
Understood. @marina-mosti working on them right now.

Reply:
Hello @marina-mosti I have to inform that the test is failling
TypeError: wrapper.findAll(...).length is not a function

      54 |       })
      55 | 
    > 56 |       expect(wrapper.findAll('.schema-row + .custom-class-a').length()).toEqual(2)
         |                                                               ^
      57 |     })
      58 |   })
      59 | 
Will review the syntax and let you know.

Reply:
That's odd... https://vue-test-utils.vuejs.org/api/wrapper/#findall

Reply:
I think its a problem with the selector, should probably be `.schema-row.custom-class-a`

Reply:
I have found this warning
https://vue-test-utils.vuejs.org/api/wrapper/#findall
Deprecation warning

Using findAll to search for Components is deprecated and will be removed. Use findAllComponents instead.

Reply:
You're not looking for a component, you're querying a class selector :)
`findAll` is basically doing `querySelectorAll`, which is correct. But the `+` in the CSS selector is incorrect. The classes are applied to the same element, not to a sybling

Reply:
There is something wrong with VTU
      console.log(wrapper.html()) // Return HTML and clearly there are 2 div with both classes
      // https://vue-test-utils.vuejs.org/api/selectors.html
      console.log(wrapper.findAll('.schema-row').length); // Returns 2
      console.log(wrapper.findAll('.custom-class-a').length);// Returns 2
      console.log(wrapper.findAll('.schema-row + .custom-class-a').length); // Returns 1
      console.log(wrapper.findAll('.schema-row ~ .custom-class-a').length); // Returns 1

Reply:
As I said, it should be `.schema-row.custom-class-a`
Please update the test with the correct selector